main/test/flix/Test.Handler.ZeroShot.AtomicInteger.flix

```diff
@@ -0,0 +1,169 @@
+mod Test.Handler.ZeroShot.AtomicInteger {
+
+    enum AtomicInt(##java.util.concurrent.atomic.AtomicInteger)
```

Comment:
haha! Smart.

But, we actually need the concrete type to test that it works with java types.

You can use a type alias though, `type alias AtomicInt = ##...`.

Reply:
But, on a second thought, this made me realize that we will want a `Test.Handler.ZeroShot.Option.flix` which uses the Flix `Option` type (let us say with an Int32 inside).

Reply:
Lets call the file Test.Handler.ZeroShot.Enum.flix, thanks

Reply:
What about tuples and records?

Reply:
No that's OK. They are handled like enums more or less.